### ISSUE TITLE
Please promote flasher page link in your CYD projects

### ISSUE THREAD

**Issue by makergamer** [ATOMNFT/CM-Box#2]
tl;dr - Please update both projects to have an obvious link to your flasher -- which worked for me.

I could not get my NIB CYD v1 to flash properly with the internet promoted https://github.com/Fr4nkFletcher/ESP32-Marauder-Cheap-Yellow-Display/ flasher https://fr4nkfletcher.github.io/Adafruit_WebSerial_ESPTool/ .

Over on the CYD Discord, a search for "marauder" had your flasher link appear. It worked as your has more verbose diagnostic details (reset didn't work, I had to plug in while holding boot -- which made sense afterwards. I'm not used to web flashers.)

Please promote your https://atomnft.github.io/CM-Box/flash0.html on both your projects as looking through their frontpage README.md and project descriptions didn't have it. I only had found it via the Discord channel.
* https://github.com/ATOMNFT/CM-Box
* https://github.com/ATOMNFT/CYD-ESP32Marauder

**Comment by ATOMNFT** [comment]
> tl;dr - Please update both projects to have an obvious link to your flasher -- which worked for me.
> 
> I could not get my NIB CYD v1 to flash properly with the internet promoted https://github.com/Fr4nkFletcher/ESP32-Marauder-Cheap-Yellow-Display/ flasher https://fr4nkfletcher.github.io/Adafruit_WebSerial_ESPTool/ .
> 
> Over on the CYD Discord, a search for "marauder" had your flasher link appear. It worked as your has more verbose diagnostic details (reset didn't work, I had to plug in while holding boot -- which made sense afterwards. I'm not used to web flashers.)
> 
> Please promote your https://atomnft.github.io/CM-Box/flash0.html on both your projects as looking through their frontpage README.md and project descriptions didn't have it. I only had found it via the Discord channel.
> 
> * https://github.com/ATOMNFT/CM-Box
> * https://github.com/ATOMNFT/CYD-ESP32Marauder

Will do, and thank you so much for the kind words and advice.

**Comment by ATOMNFT** [comment]
> tl;dr - Please update both projects to have an obvious link to your flasher -- which worked for me.
> 
> I could not get my NIB CYD v1 to flash properly with the internet promoted https://github.com/Fr4nkFletcher/ESP32-Marauder-Cheap-Yellow-Display/ flasher https://fr4nkfletcher.github.io/Adafruit_WebSerial_ESPTool/ .
> 
> Over on the CYD Discord, a search for "marauder" had your flasher link appear. It worked as your has more verbose diagnostic details (reset didn't work, I had to plug in while holding boot -- which made sense afterwards. I'm not used to web flashers.)
> 
> Please promote your https://atomnft.github.io/CM-Box/flash0.html on both your projects as looking through their frontpage README.md and project descriptions didn't have it. I only had found it via the Discord channel.
> 
> * https://github.com/ATOMNFT/CM-Box
> * https://github.com/ATOMNFT/CYD-ESP32Marauder

Tool was also update with corrected files. Display was off when using menu after a ap scan. But it's fixed now. Thanks again.
